qa-tests-backend/Makefile

```diff
@@ -2,6 +2,12 @@
 all: style test
 
 GRADLE := ./gradlew
+fail_fast = ""
```

Comment:
A few nitpicks.
1. Why do you need `fail_fast`?
2. Why not reuse `GRADLE` variables in commands below: `GRADLE_TEST := $(GRADLE) test --fail-fast`?
3. Instead of overriding `GRADLE_TEST` command, I suggest to leave it as `$(GRADLE) test` at all times, but introduce `GRADLE_TEST_ARGS` that is set to `--fail-fast` on flag and empty string without it.

Reply:
Fixed